tests/integration/targets/sysctl/tasks/main.yml

```
@@ -310,3 +316,75 @@
         that:
           - sysctl_invalid_set1 is failed
           - "'vm.mmap_rnd_bits' not in sysctl_invalid_conf_content.stdout"
+
+# Test file permissions for sysctl_file
+- name: Test to set file system permissions
+  when:
+    - ansible_facts.virtualization_type != 'docker'
+    - ansible_facts.distribution == 'RedHat'
```

why just RedHat ? Is it not supposed to work on any Linux distribution ?
```suggestion
    - ansible_facts.system == 'Linux'
```

---

done :)